index.js

```diff
@@ -136,9 +146,34 @@ instructions:
       or anything like that.
 `;
 
-const removeColorsFormat = winston.format((info) => {
-    info.message = stripAnsi(info.message);
-    return info;
+const testPlanSchema = z.object({
+    arrayOfSpecs: z.array(z.string()),
+});
+
+const actionStepSchema = z.object({
+    planningThoughtAboutTheActionIWillTake: z.string(),
+    action: z.object({
```

Comment:
Could you make the action type a discriminate union? That way you could have stricter types for each action type.

Reply:
Yup, let me open an issue for this

Reply:
https://github.com/zachblume/autospec/issues/23